sympy/ntheory/factor_.py

```diff
@@ -2201,3 +2201,29 @@ def is_deficient(n):
     if is_perfect(n):
         return False
     return bool(abundance(n) < 0)
+
+
+def is_amicable(n, N):
+    """Returns True if the pair ``(n, N)`` is a Amicable Number, else False.
+
+    Amicable numbers are two different numbers so related that the sum of the proper divisors of each is equal to the other number.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import divisor_sigma
+    >>> is_amicable(3259, 4576)
+    False
+    >>> is_amicable(5020, 5564)
+    True
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Amicable_numbers
+
+    """
+    if((divisor_sigma(n, 1)) - n == N and (divisor_sigma(N, 1)) - N == n ):
```

Comment:
This can be reduced to a single line: `return ...`.

Also, some shortcuts could be added before that line:
```python
if n%2 != N%2:
    if n%2:
        n, N =N, n
    if not (is_square(n) of is_square(n//2)):
        return False
    if not is_square(N):
        return False
if igcd(n, N) == 1 and n*N < 10**67:
    return False
```

ref: https://en.wikipedia.org/wiki/Amicable_numbers

Reply:
@smichr I have used below code earlier locally and it works find
```
def divsum(x): 
    sum = 1
    for i in range(2, x): 
        if x % i == 0: 
            sum += i 
    return sum
    
def is_amicable(a, b): 
    if divsum(a) == b and divsum(b) == a: 
        return True
    else: 
        return False
```
but it makes long codebase should I used this

Reply:
definitely don't use -- divisor_sigma is the way to go. @Yathartha22 's code suggestion looks good.